sample/core.py

```diff
@@ -0,0 +1,91 @@
+import requests
+import helpers
+from pprint import pprint
+
+
+class WikiPage:
+    def __init__(self, *pageid):
+        self.base_url = "https://en.wikipedia.org/w/api.php"
+
+        self.payload = {
+            'action': 'query',
+            'pageids': '|'.join(str(ID) for ID in list(pageid)),
+            'format': 'json',
+        }
+
+    # Method to fetch categories
+    def get_categories(self, clshow="!hidden"):
```

Comment:
One of the motivations of this project is so that the users need not bother themselves with the details of the API. Try to use generic names wherever possible. `clshow` is very misleading who isn't familiar with the API.